marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/WriteDataViaCustomEndpointTest.java

```diff
@@ -60,8 +60,7 @@ private void thenTheExpectedDocumentExists() {
         assertEquals("apple", doc.get("input").get("fruitName").asText(), message);
         assertEquals("red", doc.get("input").get("fruitColor").asText(), message);
 
-        assertEquals("value", doc.get("workUnit").get("someString").asText(), message);
-        assertEquals(123, doc.get("workUnit").get("someNumber").asInt(), message);
-        assertEquals("hello world", doc.get("endpointState").get("someState").asText(), message);
```

Comment:
We still want to verify that if Ernie has custom endpointState, his endpoint gets access to it.